lib/plugins/EPrints/Plugin/Import/PubRouter.pm

```diff
@@ -32,7 +32,7 @@ my %namespaces =
 		'ali' => 'http://www.niso.org/schemas/ali/1.0/',
 		'dcterms' => 'http://purl.org/dc/terms/',
 		'rioxxterms' => 'http://www.rioxx.net/schema/v2.0/rioxx/',
-		'pr' => 'http://pubrouter.jisc.ac.uk/rioxxplus/',
+		'pr' => 'http://pubrouter.jisc.ac.uk/rioxxplus/2.0/',
```

Comment:
Hi Will
This should be `..../v2.0/` (not `/2.0/`)

Reply:
Oops - should be fixed in the commit below